src/llmcompressor/transformers/utils/arg_parser/model_arguments.py

```diff
@@ -44,17 +46,7 @@ class ModelArguments:
         default=None,
         metadata={"help": "Where to store the pretrained data from huggingface.co"},
     )
-    use_fast_tokenizer: bool = field(
```

Comment:
Are we moving this anywhere?

Reply:
This arg is unused. Transformers doesn't have a coherent pattern w.r.t. handling fast versions of processors atm, so we replaced it with a try/catch where it tries to load the fast version by default, then falls back to the slow version otherwise.

We can consider adding this arg back to give users the ability to use slow processors through our api.